src/backend_memory.cc

```diff
@@ -96,7 +96,7 @@ BackendMemory::Create(
 
   *mem = new BackendMemory(
       manager, alloc_type, memory_type_id, reinterpret_cast<char*>(ptr),
-      byte_size);
+      byte_size, true /* owns_buffer */);
```

Comment:
This change will not be needed once constructor makes owns_buffer default to true.